web/app/templates/components/single-heuristic-detail.hbs

```diff
@@ -30,10 +30,19 @@
                             </tr></thead>
                             <tbody>
                             {{#each yarnappheuristicresult.details as |yarnappheuristicresultdetail|}}
-                                <tr>
-                                    <td>{{yarnappheuristicresultdetail.name}}</td>
-                                    <td>{{yarnappheuristicresultdetail.value}}</td>
-                                </tr>
+                                {{#if (not-empty yarnappheuristicresultdetail.details) }}
```

Comment:
In case of a failed job, the heuristic details (name, value and details) are as follows:
`Error, Stacktrace, <actual_stacktrace>`

It would be good to check if yarnappheuristicresultdetail.name equals "Error" rather than checking if details is non-empty. In the future if we plan to include details for other heuristics then this can break.